collector/bonding_linux.go

```diff
@@ -67,6 +67,9 @@ func readBondingStats(root string) (status map[string][2]int, err error) {
 	status = map[string][2]int{}
 	masters, err := ioutil.ReadFile(path.Join(root, "bonding_masters"))
 	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
Thanks for the review.

I'll change it to the Update() method. Coming up in a sec. I'll have to drop the test now, would you have a suggestion for that?